packages/server/test/support/fixtures/projects/e2e/cypress/integration/screenshots_spec.js

```diff
@@ -302,13 +302,28 @@ describe('taking screenshots', () => {
     })
   })
 
+  it('can pass overwrite option to replace existing filename', () => {
```

Comment:
it can be helpful to add a link to the github issue that a test covers, in case the `git blame` gets murky for a particular test down the line:

```suggestion
  // @see https://github.com/cypress-io/cypress/issues/7955
  it('can pass overwrite option to replace existing filename', () => {
```